README.md

```diff
@@ -7,23 +7,35 @@ Grouplogger is a specialized Stackdriver logging client for writing groups of lo
 ([Stackdriver documentation: Viewing related request log entries](https://cloud.google.com/appengine/docs/flexible/go/writing-application-logs#related-app-logs))
 
 ```go
-var r *http.Request
-var ctx *context.Context
+// Stackdriver requires a non-nil http.Request.
+u, err := url.Parse("http://notareal.website/search?q=hihihi")
+if err != nil {
+	log.Fatal(err)
+}
+req := &http.Request{
+	Header: http.Header{
+		"X-Cloud-Trace-Context": []string{"your-trace-id"},
+	},
+	URL: u,
+}
 
-cli, err := NewClient(ctx, "parent-id")
+cli, err := grouplogger.NewClient(context.Background(), "your-project-id")
```

Comment:
Contexts are a *little* weird with GCP––in GAE Standard, contexts besides those [generated with `appengine.NewContext()`](https://godoc.org/google.golang.org/appengine#NewContext) are rejected.

I agree that adding context generation makes the code more readable, but to pull the `context` call into its own line would be consistent with the [equivalent Stackdriver docs](https://godoc.org/cloud.google.com/go/logging#hdr-Creating_a_Client) and might draw a little more attention to the possibility that getting a context might be more complicated:

```go
ctx := context.Background()
cli, err := grouplogger.NewClient(ctx, "your-project-id")
```

Minor, though.

Reply:
👍